bin/setup.sh

```diff
@@ -1,90 +1,94 @@
+	case "$clean_database" in 
+		y|Y ) new_database
+		n|N ) break;;
+		* ) new_database
+	esac
+
 else
-	echo "WordPress config file not found. Installing..."
+	echo "New instalation..."
 	docker-compose exec --user www-data phpfpm wp core download
 	docker-compose exec --user www-data phpfpm wp core config --dbhost=mysql --dbname=wordpress --dbuser=root --dbpass=password
 
-# <<PHP
-#   define( 'WP_DEBUG', true );
-#   define( 'WP_DEBUG_LOG', true );
-#   define( 'WP_DEBUG_DISPLAY', false );
-#   @ini_set( 'display_errors', 0 );
-#   define( 'SAVEQUERIES', false );
-#   define( 'JETPACK_DEV_DEBUG', true );
-#   define( 'WP_CACHE', true );
-#   define( 'WP_CACHE_KEY_SALT', '$WP_CACHE_KEY_SALT' );
-#   define( 'WP_ENV', 'development' );
-# PHP
-
-  if [ $MULTISITE -eq 0 ]; then
-    echo " * Setting up multisite \"$TITLE\" at $URL"
-    docker-compose exec --user www-data phpfpm wp core multisite-install --url="$URL" --title="$TITLE" --admin_user=admin --admin_password=password --admin_email="$ADMIN_EMAIL" --subdomains
-    docker-compose exec --user www-data phpfpm wp super-admin add admin
-
-  else
-    echo " * Setting up \"$TITLE\" at $URL"
-    docker-compose exec --user www-data phpfpm wp core install --url="$URL" --title="$TITLE" --admin_user=admin --admin_password=password --admin_email="$ADMIN_EMAIL"
-  fi
-
-  ## ACTIVATING COMPONENTS ##
-  echo " * Activating the default theme"
-  docker-compose exec --user www-data phpfpm wp theme activate ${THEME}
-
-  # echo " * Importing test content"
-  # docker-compose exec --user www-data phpfpm curl -OLs https://raw.githubusercontent.com/manovotny/wptest/master/wptest.xml
-  # docker-compose exec --user www-data phpfpm wp import wptest.xml --authors=create
-  # docker-compose exec --user www-data phpfpm rm wptest.xml
+	read -p "is Multisite (y/n)? default n: " is_multisite
+	case "$is_multisite" in
+		y|Y ) MULTISITE=1;;
+		n|N ) MULTISITE=0;;
+		* ) MULTISITE=0;;
+	esac
+
+	if [ $MULTISITE -eq 1 ]; then
+		echo " * Setting up multisite \"$TITLE\" at $URL"
+		docker-compose exec --user www-data phpfpm wp core multisite-install --url="$URL" --title="$TITLE" --admin_user=admin --admin_password=password --admin_email="$ADMIN_EMAIL" --subdomains
+		docker-compose exec --user www-data phpfpm wp super-admin add admin
+	else
+		echo " * Setting up \"$TITLE\" at $URL"
+		docker-compose exec --user www-data phpfpm wp core install --url="$URL" --title="$TITLE" --admin_user=admin --admin_password=password --admin_email="$ADMIN_EMAIL"
+	fi
+
 fi
 
-## UPDATING COMPONENTS ##
-echo "Updating WordPress"
-docker-compose exec --user www-data phpfpm wp core update
-docker-compose exec --user www-data phpfpm wp core update-db
+wordpress_updater() {
+	## UPDATING COMPONENTS ##
+	echo "Updating WordPress"
+	docker-compose exec --user www-data phpfpm wp core update
+	docker-compose exec --user www-data phpfpm wp core update-db
+}
+
+read -p "Update Wordpress? (y/n)? default y: " is_multisite
```

Comment:
I think WordPress should always be updated.

Reply:
Not if:

* You need to develop for a current installed version.
* Need to backtrack to test something.

Reply:
Those use cases make sense, but that's a possibility not implemented in this setup.  
Normally, you don't have an installed version in the repo(WP installation is ignored) and you don't have a place to define what version should be installed.  
You can ask for the version, with the latest as default, and only update if no version is provided.  
Another possibility is to add all WP configuration to wp-cli.yml and let the repo decide what to do.  
I'm all in for the last option. 🙂

Reply:
When one needs to develop for a specific WP version, that should be typed by hand in the setup. This process should point us to the newest version possible. One should not start new projects already in legacy mode.

Reply:
Obviously one should not start a new project with an old version of WP @RicCastelhano. 

With this PR is possible @Narayon. As long as this script doesn't update "my" WP install, I can give this script the GIT url to my project and I can have a quick replica of my production, staging or whatever site or whatever version. If this script forces the update on me... well bollocks then. 

The update should be the developers choice give this script versatility.

Reply:
For that to work, you need to have the WP installed on your repo. We don't do that and you shouldn't either.  
That's why I mentioned `wp-cli.yml` or `wp-config.php`.

Reply:
Right, I just got what you meant. 
No, we don't use WordPress it self registered on the repository. But let's leave that part aside for a bit as it is beside the point. 
Whats important here is that, no matter how a given WordPress version get's there, either composer, manual download or whatever, the developer should be able to keep that specific version and not be force fed the update.

Reply:
I know what you want but I don't agree.  
First, this setup is not ready for what you want. How can I clone a repo and install a fresh WP?
Second, if we have a configuration file, why not use it?
Remember that this repo is specific to WP-Construct. We don't need to make it generic.